content/en/docs/home/contribute/content-organization.md

```diff
@@ -128,3 +131,14 @@ Some important notes to the files in the bundles:
 
 The `SASS` source of the stylesheets for this site is stored below `src/sass` and can be built with `make sass` (note that Hugo will get `SASS` support soon, see https://github.com/gohugoio/hugo/issues/4243).
 
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+See also:
```

Comment:
I'd just get rid of this line completely. The What's Next already gets a heading due to the template.

Reply:
will do...